src/components/screens/dashboard/components/ChangeAuthView.tsx

```diff
@@ -4,17 +4,16 @@ import ContentContainer from './ContentContainer';
 import Row from '../../../general/Row';
 import Switch from '../../../general/Switch';
 import {
-  registerFingerprintAuthenticator,
-  deregisterFingerprintAuthenticator,
-  isFingerprintAuthenticatorRegistered,
-  getRegisteredAuthenticators,
+  getAllAuthenticators,
   setPreferredAuthenticator,
-} from '../../../helpers/FingerprintHelper';
+} from '../../../helpers/AuthenticatorHelper';
+import {isFingerprintAuthenticatorRegistered} from '../../../helpers/FingerprintHelper';
 import type {Types} from 'onewelcome-react-native-sdk';
 import {AuthActionTypes} from '../../../../providers/auth.actions';
 import {AuthContext} from '../../../..//providers/auth.provider';
 import {useActionSheet} from '@expo/react-native-action-sheet';
 import Button from '../../../../components/general/Button';
+import OneginiSdk from 'onewelcome-react-native-sdk';
```

Comment:
why is this oneginisdk; before in "fingerprinthelper" you were importing the onewelcomesdk; is there a difference?